core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -144,6 +145,42 @@ public static <T> T callAndReturn(Logger logger, RpcCallableThrowsIOException<T>
     }
   }
 
+  /**
+   *
+   * Calls the given method and handled exception. Exceptions are
+   * accounted for in metrics and then rethrown at the end.
+   *
+   * @param logger the logger to use for this call
+   * @param method the callable to call
+   * @param methodName the name of the method, used for metrics
+   * @param description the format string of the description, used for logging
+   * @param responseObserver gRPC response observer
+   * @param args the arguments for the description
+   * @param <T> the return type of the method
+   */
+  public static <T> void invoke(Logger logger, Supplier<T> method, String methodName,
```

Comment:
This is probably not called anymore? Not using this also means that we are skipping all the metrics. Is that ok to do?

Reply:
needed for other functions in 3.0 so save it for now 😂

Reply:
no deadcode